moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -0,0 +1,42 @@
+import MonetizationOnOutlinedIcon from "@material-ui/icons/MonetizationOnOutlined";
+import DeleteOutlineIcon from "@material-ui/icons/DeleteOutline";
```

Comment:
ok, i think it makes the most sense to use the money icon when adding, editing or deleting. that would make use consistent with tags and project edits. so please nix the trash can — my bad!